docs/reference/modules/doc-type.md

```diff
@@ -66,7 +61,7 @@ This module is meant as a base class for more specific content modules. As such,
 
 ### `allowedSchema(req)`
 
-Returns a new version of the doc type's schema containing only fields for which the current user (`req.user`) has permission to edit. That is specified by the `permission` property of the schema field, or there is no `permission` property for the field. The `req` should be an existing request object.
+Returns a new version of the doc type's schema containing only fields that the current user (`req.user`) has permission to edit. This is dictated by the `permission` property of the schema field, or if is no `permission` property for the field. The `req` should be an existing request object.
```

Comment:
"or if is no": something isn't right here grammarwise

If the schema field has a `permission` property, the user must have that specific permission, otherwise they must be able to edit the doc type.

(Maybe we should delay documenting `permission` as we don't have any sensible way to grant users individual named permissions at the moment.)

Reply:
Yeah I think we should delay that.